src/sentry/models/monitorcheckin.py

```diff
@@ -16,12 +16,22 @@
 
 class CheckInStatus:
     UNKNOWN = 0
+    """No status was passed"""
+
     OK = 1
+    """Checkin had no issues during execution"""
+
     ERROR = 2
+    """Checkin failed or otherwise had some issues"""
+
     IN_PROGRESS = 3
+    """Checkin is expected to complete"""
+
     MISSED = 4
+    """Monitor did not check in on time"""
 
     FINISHED_VALUES = (OK, ERROR)
+    """Sentient values used to indicate a monitor is finished running"""
```

Comment:
Sentinel?

Reply:
yeah it might not be quite the right word https://en.wikipedia.org/wiki/Sentinel_value